README.md

```diff
@@ -1,2 +1,54 @@
 # action-baseline
-A GitHub Action for running the OWASP ZAP Baseline scan
+A GitHub Action for running the OWASP ZAP Baseline scan. 
```

Comment:
Maybe link to the wiki page (https://github.com/zaproxy/zaproxy/wiki/ZAP-Baseline-Scan)?